app/src/main/java/io/gnosis/safe/ui/transactions/details/TransactionDetailsFragment.kt

```diff
@@ -88,6 +88,7 @@ class TransactionDetailsFragment : BaseViewBindingFragment<FragmentTransactionDe
                         updateUi(it)
                     }
                     snackbar(requireView(), R.string.rejection_successfully_submitted)
```

Comment:
will we see this confirmation snackbar? Also will it appear on top of the bottom bar?

Reply:
There is two problems with the current implementation. 
1. Sometime the snackbar doesn't appear
2. The Rejection confirmation screen goes to the details first, submits the rejection (sometimes shows the snackbar) and then jumps to the tx list

Reply:
This has been resolved.